controllers/pod_models_test.go

```diff
@@ -304,8 +305,165 @@ var _ = Describe("pod_models", func() {
 					VolumeSource: corev1.VolumeSource{ConfigMap: &corev1.ConfigMapVolumeSource{
 						LocalObjectReference: corev1.LocalObjectReference{Name: fmt.Sprintf("%s-config", cluster.Name)},
 						Items: []corev1.KeyToPath{
+							{Key: "cluster-file", Path: "fdb.cluster"},
```

Comment:
Why is this changing as part of the PR? It seems like something that always should have been there, but I don't know why it would have been passing before if that were the case.

Reply:
Ah, nevermind, I see that this is just an artifact of the way the diff is shown.